dcos/http.py

```diff
@@ -27,6 +27,34 @@ def _default_is_success(status_code):
 
     return 200 <= status_code < 300
 
+def _is_request_to_dcos(url, toml_config=None):
+    """Checks if a request is for the DC/OS cluster.
+
+    :param url: URL of the request
+    :type url: str
+    :param toml_config: cluster config to use
+    :type toml_config: Toml
+    :return: whether the request is for the cluster
+    :rtype: bool
+    """
+
+    if toml_config is None:
+        toml_config = config.get_config()
+
+    dcos_url = urlparse(config.get_config_val("core.dcos_url", toml_config))
+    cosmos_url = urlparse(
+        config.get_config_val("package.cosmos_url", toml_config))
+    parsed_url = urlparse(url)
+
+    # request should match scheme + netloc
+    def _request_match(expected_url, actual_url):
+        return expected_url.scheme == actual_url.scheme and \
+                    expected_url.netloc == actual_url.netloc
+
+    request_to_cluster = _request_match(dcos_url, parsed_url) or \
+        _request_match(cosmos_url, parsed_url)
+
+    return request_to_cluster
```

Comment:
You can probably just return the or statement directly without assigning to an intermediate variable? Otherwise, you should rename this variable to something like `is_request_to_dcos`. Up to you.